include/seqan3/search/configuration/mode.hpp

```diff
@@ -23,80 +25,90 @@
+    static constexpr detail::search_config_id id{detail::search_config_id::mode};
+};
+
+} // namespace seqan3::detail
+
+namespace seqan3::search_cfg
+{
+
+/*!\brief Configuration element to receive all hits within the error bounds.
+ * \ingroup search_configuration
  * \details
  * This configuration element can be used to determine which hits are supported.
  * Currently these modes are available:
- * | Mode                         | Behaviour                                                           |
- * |------------------------------|---------------------------------------------------------------------|
- * | seqan3::search_cfg::all      | Report all hits within error bounds.                                |
- * | seqan3::search_cfg::all_best | Report all hits with the lowest number of errors within the bounds. |
- * | seqan3::search_cfg::best     | Report one best hit (hit with lowest error) within bounds.          |
- * | seqan3::search_cfg::strata   | Report all hits within best + x errors.                             |
+ *
+ * | Mode                                        | Behaviour                                                           |
+ * |---------------------------------------------|---------------------------------------------------------------------|
+ * | seqan3::search_cfg::reporting_all_hits      | Report all hits within error bounds.                                |
+ * | seqan3::search_cfg::reporting_all_best_hits | Report all hits with the lowest number of errors within the bounds. |
+ * | seqan3::search_cfg::reporting_best_hit      | Report one best hit (hit with lowest error) within bounds.          |
+ * | seqan3::search_cfg::reporting_strata_hits   | Report all hits within best + x errors.                             |
+ *
+ * The individual configuration elements to select a search mode cannot be combined with each other.
  *
  * ### Example
  *
  * \include test/snippet/search/configuration_modes.cpp
+ *
+ * \hideinitializer
+ */
+inline constexpr detail::reporting_all_hits_tag reporting_all_hits{};
+
+/*!\brief Configuration element to receive all hits with the lowest number of errors within the bounds.
+ * \ingroup search_configuration
+ * \copydetails seqan3::search_cfg::reporting_all_hits
+ * \hideinitializer
+ */
+inline constexpr detail::reporting_all_best_hits_tag reporting_all_best_hits{};
+
+/*!\brief Configuration element to receive one best hit (with the lowest number of errors).
+ * \ingroup search_configuration
+ * \copydetails seqan3::search_cfg::reporting_all_hits
+ * \hideinitializer
+ */
+inline constexpr detail::reporting_best_hit_tag reporting_best_hit{};
+
+/*!\brief Configuration element to receive all hits with the best number of errors plus the strata value.
+ *        A strong type of underlying type `uint8_t` that represents the number or errors for strata.
+ *        All hits are found with the fewest number of errors plus 'value'.
```

Comment:
What is `value`? Is it depicted in the documentation (it is not in the old one)? If not, maybe just describe it, otherwise it feels like there is something missing in the documentation.